reference/toolstacks/php/drupal8/developing-with-drupal.md

```diff
@@ -0,0 +1,113 @@
+# Developing with Drupal
+
+## Push changes to an environment
+
+Here, we'll see how to make code changes to an environment.
+
+> **note**
+> You should never be working on the Master branch since it's supposed to be your production environment.
+
+Make sure you're on a working environment. In this example we're on the
+*sprint1* branch:
+
+```bash
+$ git branch
```

Comment:
I would have expected to read just:

```bash
git checkout sprint1
```

(and I don't like `$` signs at the beginning of shell examples - it makes copy/paste harder - but it exists in plenty of other places in our docs)